src/modules/job-exec/job-exec.h

```diff
@@ -46,17 +46,23 @@ struct jobinfo;
  *
  *   - stats:   (optional) get json object of exec implementation stats
  *
+ *   - reconfig: (optional) called if requested to reload configuration
  */
 struct exec_implementation {
     const char *name;
-    int  (*config)  (flux_t *h, int argc, char **argv);
-    void (*unload)  (void);
-    int  (*init)    (struct jobinfo *job);
-    void (*exit)    (struct jobinfo *job);
-    int  (*start)   (struct jobinfo *job);
-    int  (*kill)    (struct jobinfo *job, int signum);
-    int  (*cancel)  (struct jobinfo *job);
-    int  (*stats)   (json_t **stats);
+    int  (*config)   (flux_t *h, int argc, char **argv);
+    void (*unload)   (void);
+    int  (*init)     (struct jobinfo *job);
+    void (*exit)     (struct jobinfo *job);
+    int  (*start)    (struct jobinfo *job);
+    int  (*kill)     (struct jobinfo *job, int signum);
+    int  (*cancel)   (struct jobinfo *job);
+    int  (*stats)    (json_t **stats);
```

Comment:
Do the apparent whitespace changes here belong in a different commit?

Reply:
i think it's in the right commit, it's where I define the `reconfig` callback.  Or do you mean in general I should split it out into another commit?

Reply:
Sorry, I possibly missed something obvious, bit why did every line change?

Reply:
Ah, it's to keep the same formatting. Ok, then nevermind 🙂 Probably a good reason to avoid that style in the future, but that's originally my fault!

Reply:
yeah, just to keep everything aligned :-)